packages/nodes/emoji/src/types.ts

```diff
@@ -0,0 +1,22 @@
+import { TComboboxItem } from '@udecode/plate-combobox';
+import { IEmojiTriggeringController } from './utils';
+
+export type EmojiItemData = {
+  id: string;
+  emoji: string;
+  name: string;
+  text: string;
+};
+
+export interface CreateEmoji<TData extends EmojiItemData = EmojiItemData> {
+  (data: TComboboxItem<TData>): string;
+}
+
+export interface EmojiPluginOptions<
```

Comment:
Naming pattern: `EmojiPlugin` (`...PluginOptions` would indeed be more appropriate, will consider a refactoring at some point)

Reply:
Renamed